### PR TITLE
Remove searchHintIcon

### LINKED ISSUE #178: Apply SearchView design
## Overview (Required)
- Even if it is not as designed, I want to delete the image here.
design
image
current
image

### PR DESCRIPTION
**Pull request by bowyer-app**
## Issue
- close #178

## Overview (Required)
- 

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks 👍 
Can we delete space between arrow icon and SearchView? 🤔

**Comment by bowyer-app** [comment]
I can like this.


can i use this library?
https://github.com/MiguelCatalan/MaterialSearchView

**Comment by takahirom** [comment]
As @jmatsu  is watching, wait a moment. 🙏

**Comment by Akihiroooo** [comment]
How about SearchView approach at Droidkaigi 2017？
https://github.com/DroidKaigi/conference-app-2017/blob/2fef8eddcdd0dc425d330e4ce5bbc2802518594c/app/src/main/res/layout/view_search.xml

File: frontend/android/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
+                app:contentInsetStartWithNavigation="0dp"
```
Comment (bowyer-app) [inline frontend/android/src/main/res/layout/activity_main.xml]: https://stackoverflow.com/questions/40124611/android-actionbar-searchview-change-icon-and-remove-extra-spacing

File: frontendcomponent/androidcomponent/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -22,4 +23,8 @@
         <item name="textAppearanceBody2">@style/TextAppearance.App.Body2</item>
         <item name="textAppearanceCaption">@style/TextAppearance.App.Caption</item>
     </style>
+
+    <style name="SearchViewStyle" parent="Widget.AppCompat.SearchView">
+        <item name="searchHintIcon">@null</item>
```
Comment (bowyer-app) [inline frontendcomponent/androidcomponent/src/main/res/values/themes.xml]: https://stackoverflow.com/questions/27265196/remove-search-view-icon-from-hint-text

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt [hunk 3]
```diff
@@ -103,6 +103,7 @@ class SearchFragment : DaggerFragment() {
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
         super.onCreateOptionsMenu(menu, inflater)
         inflater.inflate(R.menu.menu_toolbar_search, menu)
+        menu.findItem(R.id.search)?.isVisible = false
```
Comment (bowyer-app) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt]: menu has Unnecessary item `R.id.search`

**Comment by bowyer-app** [comment]
@takahirom 
fixed.

**Comment by takahirom** [comment]
Essential Phone 9.0
There is title 🤔 
image

**Comment by bowyer-app** [comment]
surely.
I will see if it can be modified.

**Comment by bowyer-app** [comment]
fixed 
### Essential Phone 9.0


### Pixel3 XL 9.0

**Comment by takahirom** [comment]
I tested it 👍It's perfect 🏆
Thank you

**Pull request merged**